### ISSUE TITLE
Empty Debug Console Output

### ISSUE THREAD

**Issue by RalphLungrin** [PowerShell/vscode-powershell#49]
When running the debugger, debug console output never appears.  The DebugTest.ps1 script will run, variables can be watched, breakpoints work, etc - but no output will ever appear in the debug console.  We've attempted multiple script and launch.json configurations with no success.  The latest attempt was made with:

    launch.json
```powershell
{
	"version": "0.2.0",
	"configurations": [
		{
			"name": "PowerShell",
			"type": "PowerShell",
			"request": "launch",
			"program": "DebugTest.ps1"
		}
	]
}
```
We've tried this on a couple different installs on two different machines.

[DebugAdapter.txt](https://github.com/PowerShell/vscode-powershell/files/64687/DebugAdapter.txt)
[EditorServices.txt](https://github.com/PowerShell/vscode-powershell/files/64686/EditorServices.txt)

**Comment by daviwil** [comment]
Is this in the new 0.3.0 release?  I might be seeing this also.

**Comment by RalphLungrin** [comment]
Yes, this was with the latest 0.3.0 release, although we had the same issue
with 0.2.0 prior to updating.

**Comment by daviwil** [comment]
Thanks for the report!  I'll see if I can get a bugfix update out for this soon.

**Comment by daviwil** [comment]
Just released verion 0.3.1 of the extension which fixes this issue.  Thanks for the report!
